security_check/main.py

```diff
@@ -36,11 +36,20 @@ def _check_image(image, severity):
     parsed = _run_gcloud(['describe', full_name])
 
     unpatched = 0
-    for vuln in parsed.get('vulz_analysis', {}).get('FixesAvailable', []):
+    for vuln in parsed.get('vulz_analysis', []):
```

Comment:
nit: any reason you changed this to default to an empty list instead of an empty dict? (since you're later calling `vuln.get()` on the non-default values it's a little misleading, don't think it makes a difference though).